enclave/cert.c

```diff
@@ -649,6 +654,55 @@ oe_result_t oe_cert_read_pem(
     return result;
 }
 
+oe_result_t oe_cert_read_der(
+    oe_cert_t* cert,
+    const void* der_data,
+    size_t der_size)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    Cert* impl = (Cert*)cert;
+    mbedtls_x509_crt* crt = NULL;
+    int rc = 0;
+
+    /* Clear the implementation */
+    if (impl)
```

Comment:
If an impl already exists, then shouldn't we be cleaning up contents of the impl rather than just doing memset?